src/windows/GeolocationProxy.js

```diff
@@ -142,8 +143,8 @@ module.exports = {
                 Windows.Devices.Geolocation.PositionAccuracy.high :
                 Windows.Devices.Geolocation.PositionAccuracy.default;
 
-        if (cordova.platformId == 'windows' && WinJS.Utilities.isPhone) {
-            // on Windows Phone 8.1 'positionchanged' event fails with error below if movementThreshold is not set
+        if (cordova.platformId == 'windows' && (WinJS.Utilities.isPhone || isWin10)) {
+            // on Windows Phone 8.1 and Windows 10 'positionchanged' event fails with error below if movementThreshold is not set
```

Comment:
We should just always set the movementThreshold property.  It exists for Windows 8 according to the documentation.